extension-localization/README.md

```diff
@@ -31,7 +31,7 @@ allprojects {
 
 // In the app build.gradle file
 dependencies {
-  implementation 'com.mapbox.extension:maps-localization:10.11.0-beta.1'
+  implementation 'com.mapbox.extension:maps-localization:10.10.1'
```

Comment:
I don't think we want to use older minor.patch 10.10.1 version then the existing 10.11.0-beta/rc family, actually. cc @mapbox/maps-android to discuss

Reply:
Makes sense in the release branch https://github.com/mapbox/mapbox-maps-android/pull/1961/files but not in main

Reply:
cc @SevaZhukov since it's generated by release train

Reply:
https://github.com/mapbox/mapbox-maps-android/pull/1966

Reply:
https://github.com/mapbox/mapbox-maps-android/pull/1966

Reply:
yes, patch release version shouldn't update in main, only in release branch